### PR TITLE
[Sprint 4] Feature book detail logic #2

### PR DESCRIPTION
**Pull request by tranminhtri911**
- https://dev.framgia.com/redmine/issues/137461
- Feature book detail logic #2 
- Effect : None

### CODE REVIEW

File: app/src/main/java/com/framgia/fbook/screen/bookdetail/BookDetailActivity.kt [hunk 1]
```diff
@@ -94,14 +119,64 @@ class BookDetailActivity : BaseActivity(), BookDetailContract.ViewModel, ItemOwn
...
+      for (user in users.iterator()) {
+        if (user.id == mUserId)
+          userWaitingThisBook = true
+        break
+      }
+    }
+    if (userWaitingThisBook) {
+      return mIsUserWaitingThisBook.set(true)
+    }
+    return mIsUserWaitingThisBook.set(false)
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/bookdetail/BookDetailActivity.kt]: change mIsUserWaitingThisBook.set(userWaitingThisBook)

File: app/src/main/java/com/framgia/fbook/screen/bookdetail/BookDetailActivity.kt [hunk 2]
```diff
@@ -94,14 +119,64 @@ class BookDetailActivity : BaseActivity(), BookDetailContract.ViewModel, ItemOwn
...
+      mOwners.addAll(owners)
+      for (owner in owners.iterator()) {
+        if (owner.id == mUserId) {
+          userHaveThisBook = true
+          break
+        }
+      }
+    }
+    if (userHaveThisBook) {
+      return mIsUserHaveThisBook.set(true)
```
Comment (HyperionsThs) [inline app/src/main/java/com/framgia/fbook/screen/bookdetail/BookDetailActivity.kt]: change to mIsUserHaveThisBook.set(userHaveThisBook)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -50,6 +50,9 @@
     <string name="inform">Inform</string>
     <string name="edit_category_favorite">Edit Category Favorite</string>
     <string name="update">Update</string>
+    <string name="are_you_sure_remove_owner_this_book">Are you sure remove owner this book?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.are_you_sure_remove_owner_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 4]
```diff
@@ -40,6 +40,9 @@
     <string name="are_u_want_to_sign_out">Bạn có chắc chắn muốn đăng xuất không?</string>
     <string name="update">Cập nhật</string>
     <string name="edit_category_favorite">Chỉnh sửa thể loại yêu thích</string>
+    <string name="are_you_sure_remove_owner_this_book">Bạn có chắc chắn xóa chủ sở hữu cuốn sách này?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.are_you_sure_remove_owner_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -50,6 +50,9 @@
...
+    <string name="are_you_sure_add_owner_this_book">Are you sure add owner this book?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.are_you_sure_add_owner_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 6]
```diff
@@ -40,6 +40,9 @@
...
+    <string name="are_you_sure_add_owner_this_book">Bạn có chắc chắn thêm chủ sở hữu cuốn sách này?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.are_you_sure_add_owner_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -50,6 +50,9 @@
...
+    <string name="do_you_want_to_read_this_book">Do you want to read this book?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.do_you_want_to_read_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values-vi/strings.xml [hunk 8]
```diff
@@ -40,6 +40,9 @@
...
+    <string name="do_you_want_to_read_this_book">Bạn có muốn đọc cuốn sách này?</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values-vi/strings.xml]: WARNING: [UnusedResources] The resource `R.string.do_you_want_to_read_this_book` appears to be unused
 Unused resources make applications larger and slow down builds.

**Pull request merged**